spec/lib/stellar/client_spec.rb

```diff
@@ -427,4 +427,140 @@
     end
   end
 
+  describe "#submit_transaction" do
```

Comment:
All the tests for submit_transaction involve operations that require memos. Can we add a test that has a mix of operations, some that require memo and others that don't, and a test that has only operations that don't require a memo and no memo is set to ensure the error is not raised in that case?